src/pages/Settings.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Slider } from "../components/Slider";
+import styled, { css } from "styled-components";
+
+const Button = styled.button`
+  background: transparent;
+  border-radius: 3px;
+  color: black;
+  margin: 0.5em 1em;
+  padding: 0.25em 1em;
+`;
+
+export const Settings = () => {
+  const [interruption, setInterruption] = useState("5");
+
+  console.log("number of questions: " + interruption);
+
+  // sets the interruption in state to n questions
+  const handleNumberClick = (number) => {
+    // e.preventDefault();
+    console.log(number + " questions has been clicked.");
+    setInterruption(number);
+  };
+
+  const NumberButton = (props) => (
+    <Button type="button"
+      onClick={() => 
+        handleNumberClick(props.number)
+      }
+    >
+     {props.number} 
+    </Button>
+  );
```

Comment:
Here it might be fine to just call the `setInterruption(number)` directly in the onClick function, instead of having an extra handler function.

```suggestion
  const NumberButton = (props) => (
    <Button type="button"
      onClick={() => 
         console.log(props.number + " questions has been clicked.");
          // sets the interruption in state to n questions
         setInterruption(props.number);
      }
    >
     {props.number} 
    </Button>
  );
```